pkgs/development/perl-modules/Bio-Ext-Align/default.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, buildPerlPackage, fetchzip }:
+
+buildPerlPackage {
+  pname = "BioExtAlign";
+  version = "1.5.1";
+  src = fetchzip {
+    # No longer maintained
+    url = "https://github.com/bioperl/bioperl-ext/archive/refs/tags/bioperl-ext-release-1-5-1.zip";
+    sha256 = "1h2l4m6r6ij11xhl41yc7f6fdqlclypvxdk61nyfj5fgmpm5jjzv";
+  };
```

```suggestion
  src = fetchzip {
    url = "https://github.com/bioperl/bioperl-ext/archive/refs/tags/bioperl-ext-release-1-5-1.zip";
    sha256 = "1h2l4m6r6ij11xhl41yc7f6fdqlclypvxdk61nyfj5fgmpm5jjzv";
  };

```
No matter if it is maintained or not, fetchFromGitHub should be used.